wodles/aws/wazuh_integration.py

```diff
@@ -132,6 +132,12 @@ def default_config(profile: str) -> dict:
             try:
                 # Get profile config dictionary
                 profile_config = {option: aws_config.get(profile, option) for option in aws_config.options(profile)}
+                region = profile_config.get('region')
+                if region:
```

Comment:
To be [PEP8](https://peps.python.org/pep-0008/#programming-recommendations) compliant

```suggestion
                if region is not None:
```